test/unittests/validation_test.cpp

```diff
@@ -455,7 +455,7 @@ TEST(validation, br_table_invalid_type)
     (func (param $x i32)
       (block $a (result i32)
         (block $b (result i64)
-          i32.const 1
+          i64.const 1
```

Comment:
This fix is needed because now it fist checks the return type on the stack against default label type before iterating over all labels and checking there consistency.
(Without the fix it's "type mismatch" error instead of "inconsistent labels")